src/components/MemberList.js

```diff
@@ -0,0 +1,36 @@
+import { Component } from '../core/component';
+import { memberStore, getNextMembersData } from '../store/memberStore';
+import Member from './Member';
+
+export default class MemberList extends Component {
+  constructor() {
+    super();
+    memberStore.subscribe('members', () => {
+      this.render();
+    });
```

Comment:
```suggestion
    memberStore.subscribe('members', this.render);
```

이렇게 하면 될까요..? 안 될까요..?!

불필요하게 익명함수를 만든 건 아닌지와, 왜 안 되는지를 확인해보세요!

Reply:
불필요하게 익명 함수를 생성하고 그때마다 불필요한 메모리를 사용하게 되는 문제가 생길 것 같네요,,,!!!
수정했습니다 감사합니다~!

https://github.com/KDT1-FE/Y_FE_JAVASCRIPT_PICTURE/pull/7/commits/1fccd82b2df538b0c422c6ac7bca54d776803da8

Reply:
멘토님 ! 이 부분은 render 내부에서 this를  참조하기에 화살표 함수를 사용해야 할 것 같습니다..!!

수정했습니다 감사합니다~!
https://github.com/KDT1-FE/Y_FE_JAVASCRIPT_PICTURE/pull/7/commits/bfb49b62260538f258476d7f3bc7267b4676dcf8

Reply:
오 네네,, 왜 안 되는지를 확인해봐달라구 리뷰를 남긴거였는데 지우셔가지구 되나 이게? 하고 확인하고 있었습니다 ㅋ쿠ㅜ 감사합니다~!